simulator/examples/primitives-rounded-rectangle.rs

```diff
@@ -0,0 +1,118 @@
+//! # Example: Rounded rectangle
+//!
+//! This example draws a `RoundedRectangle`. Click and drag to move the lower-left corner around
+//! the screen. The up/down arrow keys adjust stroke width, left/right the corner radius. Space
+//! cycles the stroke alignment between center/outside/inside.
+
+use embedded_graphics::{
+    pixelcolor::Rgb888,
+    prelude::*,
+    primitives::*,
+    style::{PrimitiveStyleBuilder, StrokeAlignment},
+};
+use embedded_graphics_simulator::{
+    OutputSettingsBuilder, SimulatorDisplay, SimulatorEvent, Window,
+};
+use sdl2::keyboard::Keycode;
+
+fn draw(
+    top_left: Point,
+    size: Size,
+    radius: Size,
+    stroke_width: u32,
+    align: StrokeAlignment,
+    display: &mut SimulatorDisplay<Rgb888>,
+) {
+    display.clear(Rgb888::BLACK).unwrap();
+
+    RoundedRectangle::with_equal_corners(Rectangle::new(top_left, size), radius)
+        .into_styled(
+            PrimitiveStyleBuilder::new()
+                .stroke_width(stroke_width)
+                .stroke_color(Rgb888::RED)
+                .stroke_alignment(align)
+                .fill_color(Rgb888::GREEN)
+                .build(),
+        )
+        .draw(display)
+        .unwrap();
+}
+
+fn main() -> Result<(), core::convert::Infallible> {
+    let mut display: SimulatorDisplay<Rgb888> = SimulatorDisplay::new(Size::new(200, 200));
+    let output_settings = OutputSettingsBuilder::new()
+        .scale(2)
+        .pixel_spacing(1)
+        .build();
+    let mut window = Window::new("Rounded rectangle debugger", &output_settings);
+
+    let top_left = Point::new(20, 20);
+
+    let mut mouse_down = false;
+
+    let mut bounding_rect = Rectangle::with_corners(top_left, Point::new(100, 100));
```

Comment:
The docs for `RoundedRectangle::new` call this the "base rectangle", so I would suggest to use the same name here.